testing/src/main/java/org/oppia/android/testing/KonfettiViewMatcher.kt

```diff
@@ -11,13 +11,18 @@ class KonfettiViewMatcher {
     /**
      * This function checks that the view being checked is a [KonfettiView] with an active confetti animation.
      *
-     * @returns a [TypeSafeMatcher] that performs matching on a view
+     * @returns a [TypeSafeMatcher] that performs matching on the view
      */
     fun hasActiveConfetti(): TypeSafeMatcher<View> {
       return ActiveConfettiMatcher()
     }
 
-    // Custom class to check the given view is a KonfettiView and isActive().
+    /**
+     * A custom class to check that the associated view is a [KonfettiView] and isActive().
+     *
+     * @returns a [TypeSafeMatcher] that checks if the given view is a [KonfettiView] that isActive with
```

Comment:
``@returns`` is only relevant for methods--classes can't actually 'return" anything. Maybe move this context to the overall documentation summary part of this doc comment?

Reply:
Also, should this actually be the returns line for ``hasActiveConfetti()`` instead?

Reply:
Yup -- done